### PR TITLE
[jsk_perception/doc] add grabcut doc

### PR DESCRIPTION
**Pull request by himlen1990**
add doc

### CODE REVIEW

**Comment by himlen1990** [comment]
add doc for #534

File: doc/jsk_perception/nodes/non_maximum_suppression.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+non_maximum_suppression in image processing (object detection)
+
+## request
+* `rect'(bounding box)
+* `threshould'
+
+## response
+* `bounding box` 
+* 'bounding box count`
```
Comment (wkentaro) [inline doc/jsk_perception/nodes/non_maximum_suppression.md]:
```
`bouding box count`
```

Comment (k-okada) [inline doc/jsk_perception/nodes/non_maximum_suppression.md]: @himlen1990 please fix this line

**Comment by k-okada** [comment]
closed via #1853

**Pull request closed without merging**